public/base.js

```diff
@@ -80,7 +80,9 @@ var a = Math.abs,
   ac = Math.acos,
   at = Math.atan,
   e = Math.E,
-  π = Math.PI;
+  π = Math.PI,
+  p = Math.pow,
+  sr = function(x) { return x * x; };
```

Comment:
Using babel [exponentiation-operator](https://github.com/rwaldron/exponentiation-operator), these can now be done as `x**y`. I believe this is shorter or at least the same length as these suggestions but maybe there is another use case for these?